elfeed-org.el

```diff
@@ -195,6 +196,41 @@ all.  Which in my opinion makes the process more traceable."
     "Load all feed settings before elfeed is started."
     (rmh-elfeed-org-process rmh-elfeed-org-files rmh-elfeed-org-tree-id)))
 
+(org-add-link-type "elfeed" 'org-elfeed-open)
+(add-hook 'org-store-link-functions 'org-elfeed-store-link)
+
+(defun org-elfeed-open (path)
```

Comment:
Would you be so kind to rename these function to make them consistent with the naming of the other functions?

```org-elfeed-open``` -> ```rmh-elfeed-org-open```
```org-elfeed-store-link``` -> ```rmh-elfeed-org-store-link```